regression/strings-smoke-tests/java_parselong/test_parselong.java

```diff
@@ -0,0 +1,39 @@
+public class test_parselong
+{
+    public static void main(String[] args)
+    {
+        if (args.length == 1) {
```

Comment:
The first four take ~6 seconds each, the last one takes ~10 seconds. I think it's a function of how long the string is.